src/app/static/src/app/store/errors/errors.ts

```diff
@@ -10,6 +10,14 @@ export interface ErrorsState {
     sendingErrorReport: boolean
 }
 
+export enum ErrorReportDefaultValue {
+    calibrate = "no associated calibrateId",
+    model = "no associated modelRunId",
+    download = "none",
+    country = "no associated country",
+    project = "no associated project"
+}
```

Comment:
Could we use a locales key word instead so that they can be translated?

Reply:
They don't really need to be translated as these values in the error reports are only seen by the Naomi support team, not by the user.